.travis.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         - codeclimate-test-reporter < coverage.txt
     # YAML alias, for settings shared across the simpler builds
     - &simple-test
-      go: 1.7.x
+      go: 1.9.x
```

Comment:
i think it needs to be just 1.9

Reply:
I was holding out for them to update and it looks like they did at some point today. Travis is now recognizing 1.9.x and passing only 1.9 to gimme. Strange that they don't have a general rule for picking up `1.Y.X` for any `Y` though.